virtualization.go

```diff
@@ -179,27 +198,50 @@ func (v *VirtualMachine) StateChangedNotify() <-chan VirtualMachineState {
 
 // CanStart returns true if the machine is in a state that can be started.
 func (v *VirtualMachine) CanStart() bool {
-	return bool(C.vmCanStart(v.Ptr(), v.dispatchQueue))
+	if macosMajorVersionLessThan(11) {
+		return false
+	} else {
+		return bool(C.vmCanStart(v.Ptr(), v.dispatchQueue))
+	}
```

Comment:
Go does not recommend return within if else block
Please fix
```suggestion
	if macosMajorVersionLessThan(11) {
		return false
	}
	return bool(C.vmCanStart(v.Ptr(), v.dispatchQueue))
```

Reply:
Fixed.